cypress/integration/chi-custom-elements/brand.spec.js

```diff
@@ -0,0 +1,54 @@
+describe('Brand', () => {
+  before(() => {
+    cy.visit('tests/custom-elements/brand.html');
+  });
+
+  beforeEach(() => {
+    ['brand-base-default', 'brand-inverse-sm', 'brand-white-md', 'brand-black-lg', 'brand-black-default', 'brand-white-xxl'].forEach((s) => {
+      cy.get(`[data-cy=${s}]`, { timeout: 5000 })
+        .should('have.class', 'hydrated')
+        .children()
+        .first()
+        .should('match', 'a.a-brand')
+        .as(s);
+    });
+  });
+
+  it('Base brand should be included inside the custom element', () => {
+    cy.get('@brand-base-default')
+      .children()
+      .first()
+      .should('have.id', 'brand-base');
```

Comment:
@srinath-koneru, Why do you test if the element has an id? This should not be part of the component.

Reply:
This is testing whether the el brand-base-default able to get hold of the base brand ```svg``` or not for that I'm using unique id's , is it not correct?